google/mozc
# Remove `WORKSPACE` support from Bazel builds

## Description
As Mozc has already migrated to bzlmod from WORKSPACE in #1002, let's remove obsolete and unmaintained WORKSPACE-related files and configs to make it clear that WORKSPACE is no longer supported to build Mozc with Bazel.

Closes #1115.

## Issue IDs
 * https://github.com/google/mozc/issues/1115

## Steps to test new behaviors (if any)
A clear and concise description about how to verify new behaviors (if any).
 - OS: All
 - Steps:
   1. Build Mozc with Bazel
   2. The build succeeds.
